humilis_sam/outputs.yaml.j2

```diff
@@ -17,7 +31,7 @@ outputs:
                 - - 'https://'
                   - Ref:
                       {% if swagger_template %}
-                      ApiGateway{{_layer.name}}{{_env.stage}}
```

Comment:
To be honest I wasn't clear about the difference between logical and physical names so I just put layer and stage name everywhere to be sure my names would be unique 😅 
I have just read about it now and yes it's way better to name it just `ApiGateway`